### PR TITLE
DEVDOCS-3498: relativize links

### PR DESCRIPTION
**Pull request by slsriehl**
# [DEVDOCS-3498](https://jira.bigcommerce.com/browse/DEVDOCS-3498)

Please review and merge when complete.

* Made local links relative
* Revised links that I knew offhand were incorrect
* Removed ToCs and converted callouts here -- can roll back those commits on request.

### CODE REVIEW

File: reference/store_content.v2.yml [hunk 1]
```diff
@@ -1152,7 +1116,14 @@ paths:
...
       operationId: updateARedirectURL
     delete:
-      description: "Deletes a *Redirect URL*.\n\n<div class=\"HubBlock--callout\">\n<div class=\"CalloutBlock--warning\">\n<div class=\"HubBlock-content\">\n\n### **Deprecated**\n\nAvoid using this API operation if possible. It will be removed in a future version.\n\nFor the most up-to-date version of this API, see [Delete Redirects v3](https://developer.bigcommerce.com/api-reference/storefront/redirects/redirects/deleteredirects) to delete a redirect URL.\n\t\n</div>\n</div>\n</div>"
+      description: |-
+        Deletes a *Redirect URL*.
+        
+        <!-- theme: warning -->
+        > #### Deprecated
+        > Avoid using this API operation if possible. It will be removed in a future version.
+        > For the most up-to-date version of this API, see [Delete Redirects v3](/api-reference/storefront/redirects/redirects/deleteredirects) to delete a redirect URL.
```
Comment (traci707) [inline reference/store_content.v2.yml:1125]:
```suggestion
        > For the most up-to-date version of this API, see [Delete Redirects v3](/api-reference/store-management/redirects/redirects/deleteredirects) to delete a redirect URL.
```

File: reference/storefront_tokens.v3.yml [hunk 2]
```diff
@@ -103,21 +97,16 @@ info:
...
     > * Unlike normal GraphQL API tokens, they are sensitive and should be treated like secrets, just as you might treat an OAuth token.
     > * Attempts to run requests using these tokens from a web browser will be rejected.
 
-    </div>
-    </div>
-    </div>
 
-    ## [Revoking tokens](https://developer.bigcommerce.com/api-reference/cart-checkout/storefront-api-token/api-token/revoketoken)
-    To [revoke a token](https://developer.bigcommerce.com/api-reference/cart-checkout/storefront-api-token/api-token/revoketoken), send a `DELETE` request to `/v3/storefront/api-token`.
+    ## [Revoking tokens](/api-reference/cart-checkout/storefront-api-token/api-token/revoketoken)
```
Comment (traci707) [inline reference/storefront_tokens.v3.yml:108]:
```suggestion
    ## [Revoking tokens](/api-reference/store-management/tokens/api-token/revoketoken)
```

File: reference/storefront_tokens.v3.yml [hunk 3]
```diff
@@ -103,21 +97,16 @@ info:
...
+    To [revoke a token](/api-reference/cart-checkout/storefront-api-token/api-token/revoketoken), send a `DELETE` request to `/v3/storefront/api-token`.
```
Comment (traci707) [inline reference/storefront_tokens.v3.yml:109]:
```suggestion
    To [revoke a token](/api-reference/store-management/tokens/api-token/revoketoken), send a `DELETE` request to `/v3/storefront/api-token`.
```

**Review by traci707 — COMMENTED** [review]
Updated broken links.

File: reference/catalog.v3.yml [hunk 4]
```diff
@@ -30,16 +25,16 @@ info:
...
+    For more information on Authenticating BigCommerce APIs, see: [Authentication](/api-docs/getting-started/authentication).
 
     ## Differentiating Variants and Modifiers
     [Variants](https://support.bigcommerce.com/s/article/Product-Options-v3#variations) represent a physical product made up of [Product Option](https://support.bigcommerce.com/s/article/Product-Options-v3) choices, i.e. a large blue t-shirt. Each variant can have a unique SKU.
 
     Modifiers represent a choice a customer makes about a product that doesn't represent a physical item, i.e. text to be printed on a t-shirt. Assigning a SKU to a modifier will turn it into a variant.
 
-    See [Variant Options](https://developer.bigcommerce.com/api-docs/catalog/products-overview#variant-options) and [Modifier Options](https://developer.bigcommerce.com/api-docs/catalog/products-overview#modifier-options) for more information.
+    See [Variant Options](/api-docs/catalog/products-overview#variant-options) and [Modifier Options](/api-docs/catalog/products-overview#modifier-options) for more information.
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply reference/catalog.v3.yml:37]: tried to accept, couldn't, committed on my own.  thanks!

**Comment by slsriehl** [comment]
@traci707 all link changes accepted.

**Review by traci707 — APPROVED** [review]

**Pull request merged**